cisco_aci/datadog_checks/cisco_aci/data/conf.yaml.example

```diff
@@ -44,53 +87,203 @@ instances:
+    ##
+    ## Set HTTP or HTTPS proxies for all instances. Use the `no_proxy` list
+    ## to specify hosts that must bypass proxies.
+    ##
+    ## The SOCKS protocol is also supported like so:
+    ##
+    ##   socks5://user:pass@host:port
+    ##
+    ## Using the scheme `socks5` causes the DNS resolution to happen on the
+    ## client, rather than on the proxy server. This is in line with `curl`,
+    ## which uses the scheme to decide whether to do the DNS resolution on
+    ## the client or proxy. If you want to resolve the domains on the proxy
+    ## server, use `socks5h` as the scheme.
+    #
+    # proxy:
+    #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
+    #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    #   no_proxy:
+    #   - <HOSTNAME_1>
+    #   - <HOSTNAME_2>
+
+    ## @param skip_proxy - boolean - optional - default: false
+    ## This overrides the `skip_proxy` setting in `init_config`.
+    ##
+    ## If set to `true`, this makes the check bypass any proxy
+    ## settings enabled and attempt to reach services directly.
+    #
+    # skip_proxy: false
+
+    ## @param auth_type - string - optional - default: basic
+    ## The type of authentication to use. The available types (and related options) are:
+    ##
+    ##   - basic
+    ##     |__ username
+    ##     |__ password
+    ##   - digest
+    ##     |__ username
+    ##     |__ password
+    ##   - ntlm
+    ##     |__ ntlm_domain
+    ##     |__ password
+    ##   - kerberos
+    ##     |__ kerberos_auth
+    ##     |__ kerberos_cache
+    ##     |__ kerberos_delegate
+    ##     |__ kerberos_force_initiate
+    ##     |__ kerberos_hostname
+    ##     |__ kerberos_keytab
+    ##     |__ kerberos_principal
+    ##   - aws
+    ##     |__ aws_region
+    ##     |__ aws_host
+    ##     |__ aws_service
+    ##
+    ## The `aws` auth type relies on boto3 to automatically gather AWS credentials (e.g. from `.aws/credentials`).
+    ## Details: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials
+    #
+    # auth_type: basic
+
+    ## @param tls_verify - boolean - optional - default: true
+    ## Instructs the check to validate the TLS certificate of services.
+    #
+    # tls_verify: true
+
+    ## @param tls_use_host_header - boolean - optional - default: false
+    ## If a `Host` header is set, this enables its use for SNI (matching against the TLS certificate CN or SAN).
     #
-    # timeout: 15
+    # tls_use_host_header: false
 
-    ## @param min_collection_interval - integer - optional - default: 180
-    ## This check makes a lot of API calls
-    ## it could sometimes help to add a minimum collection interval
+    ## @param tls_ignore_warning - boolean - optional - default: false
+    ## If `tls_verify` is disabled, security warnings are logged by the check.
+    ## Disable those by setting `tls_ignore_warning` to true.
     #
-    # min_collection_interval: 180
```

Comment:
Why `min_collection_interval` was 180 ?

Reply:
Added here: https://github.com/DataDog/integrations-core/pull/3419/files#diff-7ea38c1b670da45b488904966aa2f0a1R76-R80

Maybe a typo. cc @l0k0ms

Reply:
Min collection interval was increased because the check takes over 2 minutes to run.

Reply:
@ChristineTChen It's still optional. So the intent is just to provide a higher default ?

Reply:
It was first added here: https://github.com/DataDog/integrations-core/pull/1666/files#diff-e6dff11e233f29ac289feba74aacdf24R23-R25

Reply:
Added as comment here to reflect the initial intent: https://github.com/DataDog/integrations-core/pull/6314/commits/ae4a6f49627d83457769dd7f05a946d72e0743b8

Reply:
According our docs the true default is 15 https://docs.datadoghq.com/developers/write_agent_check/?tab=agentv6v7#collection-interval. I  think the default value should be `15` and perhaps the example can be `180` so that it's known to be the recommended collection interval.

Reply:
Yes that's right, it's 15 sec: https://github.com/DataDog/datadog-agent/blob/1c76b8381a195a0b0f629011a6225e936fe1d37a/pkg/collector/check/defaults/defaults.go#L13-L15

> I think the default value should be 15 and perhaps the example can be 180 so that it's known to be the recommended collection interval.

That's not possible for now, but could be an improvement to the config spec tooling.

Reply:
It's available from this https://github.com/DataDog/integrations-core/pull/6178.

Reply:
Updated :)